core/build.gradle

```diff
@@ -17,3 +17,10 @@ plugins {
   id 'maru.kotlin-library-conventions'
   id 'java-test-fixtures'
 }
+
+dependencies {
+  implementation "org.hyperledger.besu:besu-datatypes"
```

Comment:
I didn't find where it's used. Can we remove it?

Reply:
Removed it. Needed when I had the Hash type being used in core for hashing